bigframes/ml/compose.py

```diff
@@ -115,14 +115,17 @@ def camel_to_snake(name):
             name = re.sub("(.)([A-Z][a-z]+)", r"\1_\2", name)
             return re.sub("([a-z0-9])([A-Z])", r"\1_\2", name).lower()
 
+        output_names = []
         for transform_col in bq_model._properties["transformColumns"]:
+            transform_col_dict = cast(dict, transform_col)
             # pass the columns that are not transformed
-            if "transformSql" not in transform_col:
+            if "transformSql" not in transform_col_dict:
                 continue
-            transform_sql: str = cast(dict, transform_col)["transformSql"]
+            transform_sql: str = transform_col_dict["transformSql"]
             if not transform_sql.startswith("ML."):
                 continue
 
+            output_names.append(transform_col_dict["name"])
```

Comment:
Should we also check the existence of this key in `transform_col_dict`?

Reply:
We can. But in all cases it should exist, and nothing we can do if it doesn't. Checking it still raises an internal error, I feel it makes no difference.